### PR TITLE
Fix waitForSync for non-maintainer build 3.10

### PR DESCRIPTION
**Pull request by MBkkt**
### Scope & Purpose

https://github.com/arangodb/arangodb/pull/17004

**PLEASE MERGE AFTER:** https://github.com/arangodb/arangodb/pull/16983

- [ ] :hankey: Bugfix
- [ ] :pizza: New feature
- [ ] :fire: Performance improvement
- [ ] :hammer: Refactoring/simplification

### Checklist

- [ ] Tests
  - [ ] **Regression tests**
  - [ ] C++ **Unit tests**
  - [ ] **integration tests**
  - [ ] **resilience tests**
- [ ] :book: CHANGELOG entry made
- [ ] :books: documentation written (release notes, API changes, ...)
- [ ] Backports
  - [ ] Backport for 3.10: *(Please link PR)*
  - [ ] Backport for 3.9: *(Please link PR)*
  - [ ] Backport for 3.8: *(Please link PR)*

#### Related Information

*(Please reference tickets / specification / other PRs etc)*

- [ ] Docs PR: 
- [ ] Enterprise PR:
- [ ] GitHub issue / Jira ticket:
- [ ] Design document:

### CODE REVIEW

File: CHANGELOG [hunk 1]
```diff
@@ -1,6 +1,12 @@
 v3.10.0 (XXXX-XX-XX)
 --------------------
 
+* Fix SEARCH-340 and SEARCH-341: add stats and metrics for inverted index.
+
```
Comment (jsteemann) [inline CHANGELOG:5]: Code LGTM. Are SEARCH-340 and SEARCH-341 related to this PR?

**Review by jsteemann — APPROVED** [review]
LGTM

File: CHANGELOG (hunk 1 above)
Comment (jsteemann) [reply CHANGELOG:5]: Vale.

**Review by Dronplane — APPROVED** [review]
LGTM

**Pull request merged**